routes/productRouter.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const productController = require("../controllers/productController");
+
+const router = express.Router();
+
+router.get("/", productController.getproducts);
+router.get("/detail", productController.getProductDetail);
```

Comment:
```suggestion
router.get("/:productId", productController.getProductDetail);
```
product에 대한 상세페이지를 받을때는 query parameter로 값을 받기보다는 path parameter로 받을 수 있게끔 수정해주세요~